### ISSUE TITLE
Why no xavier initializers? and a few other questions

### ISSUE THREAD

**Issue by jeffwillette** [fmu2/PyTorch-MAML#5]
Hi. Thanks for publishing this. I have always had a really hard time reproducing the results of the MAML paper and I have to get to the bottom of it for my own sanity. You seem to have followed the original repo quite carefully but I noticed you do not use xavier initializers as they do. This must have been deliberate I assume, so I am curious why you did not use them? 

Another repo (https://github.com/haebeom-lee/maml) also didn't use xavier. 

I am trying to reproduce the results while using Pytorch higher library https://github.com/facebookresearch/higher and so far I am getting pretty bad overfitting on miniimagenet (5way 1 shot train get to mid 50's accuracy but stays around 30% on test set). I have also gone through everything in the original repo many times and it all seems to be correct and also matching with yours...

Were there any other tricky parts while you were implementing this that you got stuck on?
